<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>全局设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QCheckBox" name="cBoxSelectTrans">
        <property name="text">
         <string>启用划词翻译</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelApi">
        <property name="text">
         <string>翻译API：</string>
        </property>
        <property name="buddy">
         <cstring>cBoxApi</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cBoxApi">
        <item>
         <property name="text">
          <string>百度翻译</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>有道翻译</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelRmLB">
        <property name="text">
         <string>文本替换策略：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cBoxRmLB">
        <item>
         <property name="text">
          <string>不做任何改动</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>删除所有换行符</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>删除无断句符的换行符</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cBoxAutoStart">
        <property name="text">
         <string>开机自启动</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpBaidu">
     <property name="title">
      <string>百度翻译</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBaiduAppid">
        <property name="text">
         <string>APPID：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditBaiduAppid</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditBaiduAppid"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBaiduKey">
        <property name="text">
         <string>密钥：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditBaiduKey</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditBaiduKey"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="grpYoudao">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>有道翻译</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelYoudaoAppid">
        <property name="text">
         <string>APPID：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditBaiduAppid</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditYoudaoAppid"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelYoudaoKey">
        <property name="text">
         <string>密钥：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditBaiduKey</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditYoudaoKey"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
